roles/docker/templates/daemon.json

```diff
@@ -60,7 +60,7 @@
     "tlskey": "",
     "tlsverify": true,
     "userns-remap": "",
-    "add-registry": {{  docker_additional_registries|default([]) }},
-    "blocked-registries": {{ docker_blocked_registries|default([]) }},
+    "add-registry": {{ l_docker_additional_registries | default([]) }},
+    "blocked-registries": {{ l_docker_blocked_registries|default([]) }},
     "userland-proxy-path": "/usr/libexec/docker/docker-proxy-current"
 }
```

Comment:
> invalid character 'T' looking for beginning of value\n
I think this is related to the bad json from before

> both as a flag and in the configuration file

Will fix. Sorry I missed that.